app/controllers/users_controller.rb

```diff
@@ -3,6 +3,8 @@ class UsersController < ApplicationController
   before_action :logged_in_user, only: %i(index edit update destroy)
```

Comment:
this callback not include `show` action and show action will not work because in that action, `@user` nil

Reply:
em gọi "show" từ find_user  r mà.